package.json

```diff
@@ -22,6 +22,14 @@
   },
   "homepage": "https://github.com/Ian-Stewart-Binks/foosball#readme",
   "devDependencies": {
-    "express": "^4.13.3"
+    "express": "^4.13.3",
```

Comment:
@Anirudh94 Good catch. There currently is very little thought put into what should be in `dependencies` and what should be put into `devDependencies`. I'll update you here on what we should put, and where.

React-bootstrap was a package I was using for experimentation, and should not be in this PR.